Sources_v3/Controllers/ChannelListController.swift

```diff
@@ -49,13 +49,13 @@ public class ChannelListControllerGeneric<ExtraData: ExtraDataTypes>: Controller
         )
 
     /// A type-erased delegate.
-    private(set) var anyDelegate: AnyChannelListControllerDelegate<ExtraData>? {
+    var multicastDelegate: MulticastDelegate<AnyChannelListControllerDelegate<ExtraData>> = .init() {
         didSet {
-            // Set `Controller` `ControllerStateDelegate` delegate
-            stateDelegate = anyDelegate
+            stateMulticastDelegate.mainDelegate = multicastDelegate.mainDelegate
+            stateMulticastDelegate.additionalDelegates = multicastDelegate.additionalDelegates
```

Comment:
I'm not sure that it works. So on did set it has empty values for:
`MulticastDelegate<AnyChannelListControllerDelegate<ExtraData>> = .init()`
Probably you have to update it additionally on `setDelegate` and `delegate { set {...} }`.
The same for `ChannelController` and `CurrentUserController`.

Reply:
Not possible.
`stateMulticastDelegate` is MulticastDelegate<ControllerStateDelegate>`
`multicastDelegate` is `MulticastDelegate<AnyChannelListControllerDelegate<ExtraData>>`
Polymorphysm doesn't work for type generics.